examples/LAMMPS_OPENFOAM/minimal_MD.cpp

```diff
@@ -0,0 +1,36 @@
+#include "mpi.h"
+#include <iostream>
+
+#include "cpl.h"
+#include "CPL_ndArray.h"
+
+using namespace std;
+
+int main() {
+
+    bool flag;
+    int MD_realm = 2;
+    MPI_Comm MD_COMM, CART_COMM;
+    CPL::ndArray<double> send_array, recv_array;
+
+    MPI_Init(NULL, NULL); 
+    CPL::init(MD_realm, MD_COMM);
+
+    int npxyz[3] = {1, 1, 1}; int periods[3] = {1, 1, 1}; 
+    MPI_Cart_create(MD_COMM, 3, npxyz, periods, 1, &CART_COMM);
+    double xyzL[3] = {8*2.099495, 8*5.668637, 8*2.099495}; double xyz_orig[3] = {0.0, 0.0, 0.0};
+    CPL::setup_md(CART_COMM, xyzL, xyz_orig);
+    CPL::get_arrays(&recv_array, 1, &send_array, 4);
```

Comment:
4 here works with the minimal CFD script, 3 also works if changed on the CFD side too
BUT doesn't work with the OpenFOAM socket; the socket seems to expect 3 elements only.

With 3 elements; the coupled simulation hangs:
```
Reading field p

Reading field U

Reading/calculating face flux field phi

CPLSocketFOAM: Analysing processor and mesh topology
CPLSocketFOAM: Defining new MPI Cartesian communicator
interp_BCtrue

Creating block mesh topology
OpenFOAM CPL topology initialisation complete
```
well at least the topology initialization is complete :)

Reply:
LAMMPS also expects 3 elements. looks like it also hangs when used with minimal CFD:
```
...
Lammps FixCPLForce forcetype: Velocity with args 1.0
cnstFCells: 8 8 8 
ifix 6 Class fix cplfix
WARNING: One or more dynamic groups may not be updated at correct point in timestep (../fix_group.cpp:171)
Setting up Verlet run ...
  Unit style    : lj
  Current step  : 10000
  Time step     : 0.005
```

Reply:
The number of elements have to match. LAMMPS should send 4 as it packages up the 1 density (or count of number of molecules) and 3 components of velocity (1+3=4). We'll need to adaprt this to maybe include some phase field variables and maybe temperature. For the CFD, it simply sends the 3 components of velocity field (as pressure is not used in the constraints of Nie et al 2004), although stress coupling would send 9 components of the stress tensor.